symplyphysics/conditions/electricity/maxwell_equations/faradays_law_of_induction.py

```diff
@@ -0,0 +1,64 @@
+from sympy import Eq, Derivative
+from sympy.core import Equality
+from symplyphysics import (units, Quantity, Symbol, validate_input)
+from symplyphysics.core.fields.operators import curl_operator as rotor
+from symplyphysics.core.fields.vector_field import VectorField
+from symplyphysics.core.dimensions import assert_equivalent_dimension
+
+## Description
+## Faraday's law of induction states that a change in magnetic induction generates a vortex electric field.
+
+## Law is: rot(E(x, y, z, t)) = -d(B(x, y, z, t)) / dt, where
```

Comment:
Better use `r` instead of `x, y, z`. We do not want to limit this to cartesian coordinates.